### ISSUE TITLE
Jil.DeserializationException: Expected: ", null

### ISSUE THREAD

**Issue by 93Omar** [prajjwaldimri/Genius.NET#20]
Calling GetSong with songid **6267625**
```C#
geniusClient.SongClient.GetSong(6267625);
```
throws this exception:
```
Jil.DeserializationException: Expected: ", null
```

Seems like there is something wrong with the deserialization but I can't find the error since I'm not familiar with the Jil library. I can't find a way to handle deserialization errors with this library in order to get a more "talking error".
Have you considered using the more popular Newtonsoft.Json?

**Comment by prajjwaldimri** [comment]
I used Jil because it is much more performant than other libraries. Check out the benchmarks here: https://github.com/kevin-montrose/Jil#serialization

**Comment by 93Omar** [comment]
Yes, I saw that but seems like I can't handle serialization/deserialization errors like I could do with Newtonsoft.Json.
I think I will try to map again the SongResponse model  from a example JSON response since I think there is something wrong there.

**Comment by prajjwaldimri** [comment]
Most probably the issue is that genius changed the response that their api sends when getting a Song, so it isn't able to handle some unmapped fields

**Comment by 93Omar** [comment]
Yes, I agree. I will open a PR if I will be able to solve the issue.

**Comment by prajjwaldimri** [comment]
Alright. If we couldn't solve this issue, we will switch this library to Newtonsoft.Json
